meson.build

```diff
@@ -2730,14 +2735,16 @@ public_programs += executable(
         install : true,
         install_dir : rootbindir)
 
+if conf.get('ENABLE_INITRD') == 1
 executable(
-        'systemd-volatile-root',
-        'src/volatile-root/volatile-root.c',
-        include_directories : includes,
-        link_with : [libshared],
-        install_rpath : rootlibexecdir,
-        install : true,
-        install_dir : rootlibexecdir)
+                'systemd-volatile-root',
+                'src/volatile-root/volatile-root.c',
+                include_directories : includes,
+                link_with : [libshared],
+                install_rpath : rootlibexecdir,
+                install : true,
+                install_dir : rootlibexecdir)
```

Comment:
It would be better to not conditionalize the whole stanza, but just change `install : true` to `install : get_option('initrd')`. The advantage is that it's still possible to build the executable on demand. And we avoid the ugly conditional.

Reply:
ok